docs/cis_self_assessment.md

```diff
@@ -474,7 +474,7 @@ If you are using RBAC authorization, it is generally considered reasonable to al
 Run the below command on the master node.
 
 ```bash
-/bin/ps -ef | grep kube-apiserver | grep -v grep
+/bin/ps -wwFC kube-apiserver
```

Comment:
What's the point of this change? the doc referenced doesn't use it

Reply:
It's about half as long, and doesn't require any weird double-grepping?

Reply:
No argument however we've been maintaining a level of adherence to the original doc with audit, remediation ,etc. We should continue that adherence until otherwise warranted to deviate.

Reply:
ah ok. I don't have access to the original doc so I don't know what it looks like, I just saw it as an opportunity for simplification.

Reply:
I'd simplify 90% of the commands if possible. :/

Reply:
The original doc is actually in our confluence. bryan can point at it, but yeah, i head bryan stay as aligned with the upstream audit as possible

Reply:
I want the rest of this content, so I'm just going to merge. I'll decide if i want to revert to the old or not, In the end, this is change is ok.

Also, I leared something with the ps command! :-)